sale_order_type_quotation_number/views/sale_order_type_view.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!-- Copyright 2023 Manuel Regidor <manuel.regidor@sygel.es>
+ License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="quotation_sot_sale_order_type_form_view" model="ir.ui.view">
```

Comment:
Could you also add `quotation_sequence_id` to `form` and `search` views?

Reply:
Done

Reply:
Well, I was talking about adding then to `sale.order.type` model, not `sale.order`...

Reply:
@manuelregidor review please

Reply:
@dalonsod Done